src/app/tests/TestReadInteraction.cpp

```diff
@@ -1744,7 +1739,53 @@ void TestReadInteraction::TestSubscribeUrgentWildcardEvent(nlTestSuite * apSuite
             }
             ctx.GetIOContext().DriveIO(); // at least one IO loop is guaranteed
         }
-        NL_TEST_ASSERT(apSuite, delegate.mGotEventResponse == true);
+        NL_TEST_ASSERT(apSuite, delegate.mGotEventResponse);
+
+        // Since we just sent a report, we should have our min interval timer
+        // running again.
+        NL_TEST_ASSERT(apSuite, delegate.mpReadHandler->mFlags.Has(ReadHandler::ReadHandlerFlags::HoldReport));
+        NL_TEST_ASSERT(apSuite, !delegate.mpReadHandler->IsDirty());
+        delegate.mGotEventResponse = false;
```

Comment:
Would be good to add a test where we had a second subscription that did not request urgent events, and that subscription did not actually get reports delivered to them at the min-interval just because another sub did.

This ensures we don't have any odd coupling issues between subscriptions (where one sub request urgent events somehow results in other subs being delivered reports too at the same time).

Reply:
Yes, I can do that.

Reply:
Done.